recipe/fix-ridge-instability.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/sklearn/linear_model/ridge.py b/sklearn/linear_model/ridge.py
+index f99510d..9b4aaa9 100644
+--- a/sklearn/linear_model/ridge.py
++++ b/sklearn/linear_model/ridge.py
+@@ -885,7 +885,7 @@ class _RidgeGCV(LinearModel):
+         constant_column = np.var(Q, 0) < 1.e-12
+         # detect constant columns
+         w[constant_column] = 0  # cancel the regularization for the intercept
+-        w[v == 0] = 0
```

Comment:
Rather than backporting a fix, I would be in favour of skipping the two problematic tests on OSX and merging this PR. This way 0.18.1 will be available on conda-forge while the code changes will be only minor (test changes only) compared to the 0.18.1 release available on PyPI.